index.js

```diff
@@ -11,14 +11,21 @@ if (other) {
   }
 } else
 if (osx) {
-  var regPath = '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome'
+  var osxSuffix = '/Contents/MacOS/Google Chrome'
+  var regPath = '/Applications/Google Chrome.app' + osxSuffix
   var altPath = require('userhome')(regPath.slice(1))
+  var mdFindCmd = 'mdfind \'kMDItemDisplayName == "Google Chrome" && kMDItemKind == Application\''
 
-  module.exports = fs.existsSync(regPath)
-    ? regPath
-    : altPath
+  if (fs.existsSync(regPath)) {
+    module.exports = regPath
+  } else if (fs.existsSync(altPath)) {
+    module.exports = altPath
+  } else {
+    var foundPath = require('child_process').execSync(mdFindCmd, { encoding: 'utf8' })
```

Comment:
I've written up a bunch of async browser-locaters:

- https://github.com/davidtheclark/locate-chrome
- https://github.com/davidtheclark/locate-firefox
- https://github.com/davidtheclark/locate-safari
- https://github.com/davidtheclark/locate-ie

As far as I can tell these work as well as the original `chrome-location` and `firefox-location`, but are async. They return promises and take callbacks.

@yoshuawuyts Do you suggest I close this issue, then, and open one in smokestack asking if they're interested? Or do you think I should submit a PR there integrating the async modules?